components/Act/ActDetail.js

```diff
@@ -98,6 +108,25 @@ export function ActDetail ({ act, me }) {
       </OpSectionGrid>
       <Divider />
 
+      {act.documents && act.documents.length > 0 && (
```

Comment:
There probably value in factoring this into a separate file component that takes a doc list.  it will get used in other places such as orgs and ops too.